shared/message/vision/VisualMesh.proto

```diff
@@ -25,17 +25,37 @@ import "google/protobuf/timestamp.proto";
 import "Matrix.proto";
 
 message VisualMesh {
+    /// The "almost" concentric rings of the mesh centred about the robot
     message Row {
-        float phi      = 1;
+        /// The angle out from the camera
+        float phi = 1;
+        /// The on-screen nodes in the row
         int32 segments = 2;
     }
-    uint32                    id              = 1;
-    fmat                      rays            = 2;
-    imat                      neighbourhood   = 3;
-    fmat                      classifications = 4;
-    fmat                      coordinates     = 5;
-    repeated int32            indices         = 6 [packed = true];
-    mat4                      Hcw             = 7;
-    repeated Row              mesh            = 8;
-    google.protobuf.Timestamp timestamp       = 9;
+    /// The camera which took the measurements
+    uint32 id = 1;
+    /// Unit vectors in world space at camera origin. They represent points in the
+    /// mesh (vector projected from optical axis through a pixel coordinate and on
+    /// to the observation plane). This matrix has shape `3 x N`, where N is the
+    /// number of rays in the Visual Mesh
+    fmat rays = 2;
+    /// For a given ray, this provides you with its N neighbours in the mesh. N is
+    /// defined during training. For example with the `RING6` graph, this matrix
+    /// has shape `6 x N`. The entries are indices into the other matrices in this message
```

Comment:
`6 x N` in this context is ambiguous as `N` is previously defined as the number of neighbours.
```suggestion
    /// For a given ray, this provides you with its M neighbours in the mesh. M is
    /// defined during training. For example with the `RING6` graph, `M = 6` and this matrix
    /// has shape `6 x N`. The entries are indices into the other matrices in this message
```